lib/game.rb

```diff
@@ -0,0 +1,85 @@
+class Game
+  attr_accessor :board, :player_1, :player_2, :name
+
+  # Win combinations constant
+  WIN_COMBINATIONS = [
+    [0, 1, 2], # Top row
+    [3, 4, 5], # Middle row
+    [6, 7, 8], # Bottom row
+    [0, 3, 6], # Left column
+    [1, 4, 7], # Middle column
+    [2, 5, 8], # Right column
+    [0, 4, 8], # Left Diagonal
+    [2, 4, 6]  # Right Diagonal
+  ]
+  
+  # Game methods
+  def initialize(player_1 = Players::Human.new("X", name), player_2 = Players::Human.new("O", name), board = Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+    self
+  end
+
+  def current_player
```

Comment:
Nice job using [`Integer#even?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-even-3F) here. It's a very semantic, readable method. :-)